docs/docs/usage/scaladoc/staticSite.md

```diff
@@ -29,26 +29,33 @@ index.html
 docs/getting-started.html
 ```
 
-Scaladoc can transform both files and directories (to organize your documentation into tree-like structure). By default directories has title based on file name and has empty content. There is an option to include `index.html` or `index.md` (not both) to provide both content and properties like title (see [Properties](#properties)).
+Scaladoc can transform both files and directories (to organize your documentation into a tree-like structure). By default, directories have a title based on the file name and have empty content. There is an option to include `index.html` or `index.md` (not both) to provide both content and properties like the title (see [Properties](#properties)).
```

Comment:
> There is an option
Is this a cli option? Or do you mean *It is possible to include* ?

Reply:
`It is possible to provide index pages for each section by creating `index.html` or `index.md` (not both) in the dedicated directory` is better?